Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -311,6 +311,10 @@ private static void Clean(NugetPackageIdentifier package)
             // For now, delete src.  We may use it later...
             DeleteDirectory(packageInstallDirectory + "/src");
 
+            // Since we don't automatically fix up the runtime dll platforms, remove them until we improve support
+            // for this newer feature of nuget packages.
+            DeleteDirectory(Path.Combine(packageInstallDirectory, "/runtimes"));
```

Comment:
would it be more appropriate to drop runtimes into a .External/External folder so that they aren't seen as assets but the content can still be accessed? I don't know if there are any nuget packages that only contain runtimes directories, but not being able to view that content may regress some NuGet for Unity's users experiences.

Reply:
Potentially, but I find it very unlikely because it would currently function if the only place they had a dll was in only one architecture under the runtimes folder. Otherwise they'd run into the same conflicting dll issues.